src/Shortcodes/ImageShortcodeProvider.php

```diff
@@ -71,6 +72,8 @@ public static function handle_shortcode($args, $content, $parser, $shortcode, $e
         }
 
         // Check if a resize is required
+        $width = null;
```

Comment:
Thinking it could have been interesting to read the width/height from the record when it's not provided via the argument, but that's super edge casy ... and there's bound to be a mad scientist out there that explicictely doesn't want a width or height on its image tag.